### PR TITLE
Topic/rest customization

### PR DESCRIPTION
**Pull request by cameleeck**
Recreated idea of configuring ServiceRestClient
@szpak @mariuszs @marcingrzejszczak @ArturGajowy

### CODE REVIEW

File: micro-infra-spring-base/src/test/groovy/com/ofg/infrastructure/web/resttemplate/fluent/ServiceRestClientCustomizationByNameSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package com.ofg.infrastructure.web.resttemplate.fluent
```
Comment (cameleeck) [inline micro-infra-spring-base/src/test/groovy/com/ofg/infrastructure/web/resttemplate/fluent/ServiceRestClientCustomizationByNameSpec.groovy]: It's due to merge.

File: micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/resttemplate/fluent/config/ServiceRestClientConfigurer.groovy [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package com.ofg.infrastructure.web.resttemplate.fluent.config
```
Comment (marcingrzejszczak) [inline micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/resttemplate/fluent/config/ServiceRestClientConfigurer.groovy]: pls add end lines

File: micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/resttemplate/fluent/config/ServiceRestClientConfigurer.groovy [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+    /**
+     * Configures underlaying message converters
+     * @param converters
+     */
+    void configureMessageConverters(List<HttpMessageConverter<?>> converters);
+
+    /**
+     *
+     * @param configurer
```
Comment (marcingrzejszczak) [inline micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/resttemplate/fluent/config/ServiceRestClientConfigurer.groovy]: remove this

**Comment by marcingrzejszczak** [comment]
Apart from the minor comments +1

**Comment by marcingrzejszczak** [comment]
@cameleeck why haven't you merged this yet? :P

**Comment by szpak** [comment]
Can we close it as it was already "merged"?

**Comment by marcingrzejszczak** [comment]
@cameleeck - hello? :P

**Comment by cameleeck** [comment]
You can close, it's already merged
10 kwi 2015 10:43 "Marcin Grzejszczak" <notifications@github.com>
napisał(a):

**Pull request closed without merging**